tests/snmp/test_snmp_lldp.py

```diff
@@ -35,6 +37,8 @@ def test_snmp_lldp(duthosts, enum_rand_one_per_hwsku_hostname, localhost, creds_
     (similar to lldp test)
     """
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    if duthost.is_supervisor_node():
+        pytest.skip("LLDP not supported on supervisor node")
```

Comment:
We need lldp checks for supervisor as well.
The below OIDs needs to tested.
lldpLocChassisIdSubtype', 'lldpLocChassisId', 'lldpLocSysName', 'lldpLocSysDesc'
'lldpLocManAddrLen', \
'lldpLocManAddrIfSubtype', \
'lldpLocManAddrIfId', \
'lldpLocManAddrOID']:

The checks related snmp_facts['snmp_interfaces'] are not required, but test case already handles that check as it compares with minigraph neighbor information and we dont have any neighbors for supervisor.